packages/bautajs-fastify/src/index.ts

```diff
@@ -26,10 +26,12 @@ export async function bautajsFastify(
   fastify: FastifyInstance,
   opts: BautaJSFastifyPluginOptions
 ): Promise<void>;
+
```

Comment:
@Xavier-Redondo do not commit this file please

Reply:
done